src/ApprenticePortal/SFA.DAS.ApprenticePortal/SFA.DAS.ApprenticePortal.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
 	<PackageReference Include="MediatR" />
 	<PackageReference Include="Microsoft.AspNetCore.JsonPatch" />
+	<PackageReference Include="Newtonsoft.Json" />
```

Comment:
how come these Newtonsoft references have been added to an unrelated area of the APIM-Endpoints solution?

Reply:
The solution wouldn't build without them